.github/workflows/on-release-publish.yml

```
@@ -189,14 +189,35 @@ jobs:
       signature_x64: ${{ needs.win-sigs.outputs.sig_x64 }}
       signature_arm64: ${{ needs.win-sigs.outputs.sig_arm64 }}
 
+  macos-updater:
+    needs: [setup, win-sigs]
+    if: ${{ needs.win-sigs.outputs.enabled == 'true' }}
+    uses: ./.github/workflows/macos-updater.yml
```

actionlint (1.7.12)

[error] secret "APPLE_SPARKLE_PRIVATE_KEY" is required by "./.github/workflows/macos-updater.yml" reusable workflow

(workflow-call)